### PR TITLE
Added section of crowdfunding

### PR DESCRIPTION
**Pull request by SeqLaz**
Added section of Crowdfunding

### CODE REVIEW

File: documentation/get-started/sponsor.en.md [hunk 1]
```diff
@@ -12,7 +12,10 @@ The best way to financially support QField is by subscribing to one of the month
 
 
 ## Crowdfundings
-Currently no crowdfundings are active
+
+You can be part of an exciting opportunity to be part of something truly impactful!. We believe that positive change starts with a collective effort, and we're inviting you to join us on this inspiring journey. To turn our vision into reality, we need your support through crowdfunding. Your contribution will not only help us reach our funding goal but also play a crucial role.
```
Comment (lucienicolier) [inline documentation/get-started/sponsor.en.md:16]:
```suggestion
You can be part of an exciting opportunity to be part of something truly impactful! We believe that positive change starts with a collective effort, and we're inviting you to join us on this inspiring journey. To turn our vision into reality, we need your support through crowdfunding. Your contribution will help us reach our funding goal and play a crucial role.
```

File: documentation/get-started/sponsor.en.md [hunk 2]
```diff
@@ -12,7 +12,10 @@ The best way to financially support QField is by subscribing to one of the month
 
 
 ## Crowdfundings
-Currently no crowdfundings are active
+
+You can be part of an exciting opportunity to be part of something truly impactful! We believe that positive change starts with a collective effort, and we're inviting you to join us on this inspiring journey. To turn our vision into reality, we need your support through crowdfunding. Your contribution will help us reach our funding goal and play a crucial role.
```
Comment (suricactus) [inline documentation/get-started/sponsor.en.md:16]:
```suggestion
You can be part of an exciting opportunity to be part of something truly impactful! We believe that positive change starts with a collective effort, and we are inviting you to join us on this inspiring journey. To turn our vision into reality, we need your support through crowdfunding. Your contribution will help us reach our funding goal and play a crucial role.
```

Reads a bit better IMO.

Comment (m-kuhn) [reply documentation/get-started/sponsor.en.md:16]: Also the initial double "be part of" could be deduplicated

File: documentation/get-started/sponsor.en.md [hunk 3]
```diff
@@ -13,7 +13,7 @@ The best way to financially support QField is by subscribing to one of the month
 
 ## Crowdfundings
 
-You can be part of an exciting opportunity to be part of something truly impactful! We believe that positive change starts with a collective effort, and we are inviting you to join us on this inspiring journey. To turn our vision into reality, we need your support through crowdfunding. Your contribution will help us reach our funding goal and play a crucial role.
+You can be part of an exciting opportunity of something truly impactful! We believe that positive change starts with a collective effort, and we are inviting you to join us on this inspiring journey. To turn our vision into reality, we need your support through crowdfunding. Your contribution will help us reach our funding goal and play a crucial role.
```
Comment (m-kuhn) [inline documentation/get-started/sponsor.en.md:16]:
```suggestion
You can be part of an exciting opportunity of something truly impactful! We believe that positive change starts with a collective effort, and we are inviting you to join us on this inspiring journey. To turn this vision into reality, we need your support through crowdfunding. Your contribution will help us reach our funding goal and play a crucial role.
```

Tiny change to make this vision include the client too

**Review by ZsanettMed — APPROVED** [review]
Great! Thank you very much!

**Comment by m-kuhn** [comment]
@ZsanettMed there was still an open discussion/proposal above. Can you make sure this is taken into account?

File: documentation/get-started/sponsor.en.md (hunk 3 above)
Comment (SeqLaz) [reply documentation/get-started/sponsor.en.md:16]: I will make a new commit with the changes, but I will need to be approved after it! @ZsanettMed, @m-kuhn

**Pull request merged**